RallyHereDeveloperAPI/Source/RallyHereDeveloperAPI/Public/RallyHereDeveloperAPIAuthContext.h

```diff
@@ -48,6 +49,7 @@ struct RALLYHEREDEVELOPERAPI_API FAuthContext : TSharedFromThis<FAuthContext>
     bool AddBearerToken(const FHttpRequestRef& HttpRequest) const;
     bool AddBearerToken(const FHttpRequestPtr& HttpRequest) const;
 
+    FString GetClientId() const;
```

Comment:
Preferably this would return the actual TOptional<FString>, so that when it is not set, it can return an empty optional rather than an empty string.